app-engine/java/src/com/google/cloud/tools/intellij/appengine/java/sdk/CloudSdkService.java

```diff
@@ -25,7 +25,8 @@
 
   /** Shortcut for getting currently active implementation of {@link CloudSdkService}. */
   static CloudSdkService getInstance() {
-    return ServiceManager.getService(CloudSdkServiceManager.class).getCloudSdkService();
+    CloudSdkServiceManager service = ServiceManager.getService(CloudSdkServiceManager.class);
+    return service != null ? service.getCloudSdkService() : null;
```

Comment:
Could we add a small comment on why this could be null? Not quite obvious since everything is there according to the code and configs.

Reply:
sure